spring-integration-core/src/main/java/org/springframework/integration/endpoint/MessageProducerSupport.java

```diff
@@ -99,9 +117,6 @@ protected void onInit() {
 	protected void doStart() {
 		Assert.state(this.outputChannel != null || StringUtils.hasText(this.outputChannelName),
 				"'outputChannel' or 'outputChannelName' is required");
```

Comment:
M-m-m. Yeah... Makes sense.
So, this should be duplicated in the `getOutputChannel()`. Because we might not start the component, but the `getOutputChannel()` is called anyway in the `GraphServer.afterSingletonsInstantiated()`.